### ISSUE TITLE
How to remove an incomplete upload?

### ISSUE THREAD

**Issue by yunshi-stacy** [mapbox/mapbox-cli-py#90]
And I cannot add a new one

**Comment by perrygeo** [comment]
The upload consists of two steps: first getting the data to s3, second alerting the uploads API of to begin the processing. Do you happen to know which part it's failing on? What steps did you take which lead to the incomplete state? What is the error message?

The uploads API allows you to list, view status and delete an upload. Currently this command line interface does expose that functionality but if you are comfortable working in Python, the [mapbox python SDK](https://github.com/mapbox/mapbox-sdk-py/blob/master/docs/uploads.md) should allow you to get the status of the upload, delete it and try again. Let me know if what you find.

**Comment by yunshi-stacy** [comment]
Yes, I know. I have created the upload task, but it is incomplete forever.

{"message":"upload is incomplete”}

always in my terminal

**Comment by mapsam** [comment]
Hi @yunshi-stacy - this is likely related to an upload being stuck in our pipeline and not updating your account with an error message. This happens from time to time. If you're still seeing this error message for your specific upload, I recommend [reaching out to support](https://www.mapbox.com/contact/support/) so we can work with your Mapbox account.
